speech/snippets/multi_region_test.py

```diff
@@ -15,11 +15,15 @@
 
 from google.api_core.retry import Retry
 
+import pytest
+
 import multi_region
 
 
 @Retry()
-def test_multi_region(capsys):
+def test_multi_region(
+    capsys: pytest.CaptureFixture
+) -> None:
     multi_region.sync_recognize_with_multi_region_gcs()
     out, _ = capsys.readouterr()
```

Comment:
Similarly to the above comment - we should redo the sample to return a value and not use output parsing in the test